custom_components/moonraker/__init__.py

```diff
@@ -35,8 +35,11 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
         hass.data.setdefault(DOMAIN, {})
 
     url = entry.data.get(CONF_URL)
+    port = entry.data.get(CONF_PORT)
 
-    api = MoonrakerApiClient(url, async_get_clientsession(hass, verify_ssl=False))
+    api = MoonrakerApiClient(
```

Comment:
Do we convert the port to a int, because I only see it as a string, and looks like we want a int

Reply:
It's converted by pythonmagie when calling MoonrakerApiClient()